recipes/cgal/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+mkdir -p build && cd build
+
+# needs qt5 for imageio
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DWITH_CGAL_ImageIO=OFF -DWITH_CGAL_Qt5=OFF \
+  ..
+
+make install
+
+
+# language bindings are in a separate repo without releases
+git clone https://github.com/CGAL/cgal-swig-bindings.git
```

Comment:
Simple is good. If we went ahead with it is as it is, does that make breaking it up later harder?